scripts/compile-ajv-validators.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable */
+
+const fs = require('fs')
+const path = require('path')
+const Ajv = require('ajv')
+const standaloneCode = require('ajv/dist/standalone').default
+const addFormats = require('ajv-formats')
+const schema = require('@uniswap/token-lists/dist/tokenlist.schema.json')
+
+const ajv = new Ajv({ code: { source: true, esm: true } })
+addFormats(ajv)
+const validate = ajv.compile(schema)
+let moduleCode = standaloneCode(ajv, validate)
```

Comment:
I'd rather these be renamed to disambiguate, instead of tacking `2` onto the latter:
- `validateTokenList`, `validateTokenListCode` or `tokenListValidationCode`
- `validateTokens`, `validateTokensCode` or `tokensValidationCode`

Reply:
agree, will rename